subprojects/CLI11.wrap

```diff
@@ -0,0 +1,3 @@
+[wrap-git]
+url = https://github.com/CLIUtils/CLI11
+revision = tags/v2.1.2
```

Comment:
I recommend you instead use the wrap file installed by:

```
meson wrap install cli11
```

It looks like this:
```[wrap-file]
directory = CLI11-2.2.0
source_url = https://github.com/CLIUtils/CLI11/archive/refs/tags/v2.2.0.tar.gz
source_filename = CLI11-2.2.0.tar.gz
source_hash = d60440dc4d43255f872d174e416705f56ba40589f6eb07727f76376fb8378fd6

[provide]
cli11 = CLI11_dep
```

It works even without git, it securely checksums the external resource...